plugins/SSH.koplugin/main.lua

```diff
@@ -58,7 +61,7 @@ function SSH:start()
         local info = InfoMessage:new{
                 timeout = 5,
                 text = string.format("%s %s \n %s",
-                    "SSH port: ", self.SSH_port,
+                    _("SSH port: "), self.SSH_port,
```

Comment:
That doesn't solve the positioning problem. (E.g., in RTL) :-)

Reply:
I don't understand what you mean. RTL? Right to Left?

Reply:
Yes, internationalization should use "blabla %1", for example so it can be freely positioned as "%1 albalb".